viam-cartographer/src/carto_facade/util.h

```diff
@@ -0,0 +1,48 @@
+// This is an experimental integration of cartographer into RDK.
+#ifndef CARTO_FACADE_UTIL_H
```

Comment:
These won't be used until later, but I figured it would be good to copy them over to the carto_facade directory while I did the same for io (which was needed to complete IOInit)